2-dom-manipulation/form.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const NAME_V = /^(([A-Za-z\s])+)$/;
+const EMAIL_V = /([a-z0-9]+[_az0-9\.-]*[a-z0-9]+)@([a-z0-9-]+(?:\.[a-z0-9-]+)*\.[a-z]{2,4})/;
+const PHONE_V = /[0-9]{2,}/;
+
+var htmlForm = document.querySelector('#form');
+htmlForm.addEventListener('submit', validateForm);
+var finalMessages = document.querySelector('#final-information');
+
+function validateForm(event) {
+    event.preventDefault();
+    let form = {
+        name: document.querySelector('#name'),
+        email: document.querySelector('#email'),
+        phone: document.querySelector('#phonen'),
+        courses: document.querySelectorAll('input[type="checkbox"]'),
+        message: document.querySelector('#texta')
+    };
+    printInHTML(form.name, form.email, form.phone, form.courses, form.message);
+}
+
+function validateName(nameToValidate) {
+    if (nameToValidate.value == '') {
+        nameToValidate.classList.add('name-error');
+        return 'Please fill the name field !';
+    }
+    else if (!NAME_V.test(nameToValidate.value)) {
+        nameToValidate.classList.add('name-error');
+        return 'Please write a valid name, not numbers or special characters !';
+    }
+    nameToValidate.classList.remove('name-error');
+    return nameToValidate.value;
+}
```

Comment:
you can change the name of the parameter to `name`. With the name of the function, we already know that it's validating.